src/hooks/use-import-export.tsx

```diff
@@ -121,6 +133,11 @@ export const ImportExportProvider = ( { children }: { children: React.ReactNode
 		} ) );
 	}, [] );
 
+	const isSiteImporting = useCallback(
```

Comment:
Nice! We could probably use it in https://github.com/Automattic/studio/blob/072db9b36177e9e7e08e58acea17bc6b7fab13a8/src/components/content-tab-import-export.tsx#L25 but we can refactor this in a different PR